src/Pages/Adopt.jsx

```diff
@@ -679,6 +750,411 @@ export const Adopt = () => {
+                        "& .MuiFormLabel-root.MuiInputLabel-shrink": { whiteSpace: "nowrap" },
+                      }}
+                      variant="outlined"
+                      multiline
+                      minRows={4}
+                      {...register("smokers_explanation", {
+                        required: "Second-hand smoke plan is required",
+                      })}
+                      error={!!errors.smokers_explanation?.message}
+                      helperText={errors.smokers_explanation?.message}
+                    />
+                  </Fade>
+                )}
+                <FormLabel
+                  id="other-pets-bool"
+                  sx={{ fontWeight: "bold" }}
+                >
+                  Do you currently have other pets living in your home?
+                </FormLabel>
+                <Controller
+                  control={control}
+                  name="other_pets_in_home"
+                  render={({ field: { onChange, value, name } }) => (
+                    <RadioGroup
+                      aria-labelledby="other-pets-bool"
+                      value={value}
+                      name={name}
+                      onChange={(event) => {
+                        clearErrors("what_other_pets");
+                        onChange(event);
+                      }}
+                    >
+                      <FormControlLabel
+                        value="Yes"
+                        control={<Radio />}
+                        label="Yes"
+                      />
+                      <FormControlLabel
+                        value="No"
+                        control={<Radio />}
+                        label="No"
+                      />
+                    </RadioGroup>
+                  )}
+                />
+                {watchOtherPets === "Yes" && (
+                  <Fade
+                    in={true}
+                    timeout={250}
+                  >
+                    <TextField
+                      id="what-other-pets"
+                      label="If yes, please list species and how many"
+                      variant="outlined"
+                      {...register("what_other_pets", {
+                        required: "Other pets information is required",
+                      })}
+                      error={!!errors.what_other_pets?.message}
+                      helperText={errors.what_other_pets?.message}
+                    />
+                  </Fade>
+                )}
+              </Stack>
+              <Typography
+                variant="h3"
+                sx={{ mb: 2, mt: 4 }}
+              >
+                Bird Care
+              </Typography>
+              <Stack spacing={2}>
+                <TextField
+                  id="what-supp-info"
+                  label="What sources of information do you use to supplement your knowledge of avian care?"
+                  sx={{
```

Comment:
This is a fabulous solution that actually makes the input more mobile-friendly! I love it so much that I want you to put this object in a constants file somewhere so we can import it into all the forms and use it on all the inputs that have really long labels.

Reply:
Check it out... 
Your solution on mobile:
<img width="261" alt="Screenshot 2024-01-05 at 4 51 09 PM" src="https://github.com/allie-rae/rescue-the-birds/assets/48700332/7957732e-dea5-4f80-9182-f34a7e8ca514">

Old solution on mobile:
<img width="259" alt="Screenshot 2024-01-05 at 4 51 26 PM" src="https://github.com/allie-rae/rescue-the-birds/assets/48700332/29291be6-3069-4ad7-8712-4ef441259aeb">

Reply:
In my revision, I allowed multi-line labels on several more inputs that need it when on mobile. All labels can now be seen in their entirety in mobile as in full view. However, for single-line inputs this now creates  a slightly ugly spillover for a few inputs. A simple media query can solve this so when in mobile modes, we can expand certain single-line inputs to two lines to accomdate the label, but I figured we can tackle that as a separate issue.

![Screenshot 2024-01-06 121718](https://github.com/allie-rae/rescue-the-birds/assets/109903708/bf353732-4836-4c99-9f8a-e20960a9c5bc)

Reply:
I am fine with tackling it as a separate issue. ✅ Make sure to make the issue so we don't forget!